configure-buildfarm/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+# The platfroms that are supported by the self-managed build farm. 
+# Valid platforms are 'android' and 'ios'. Multiple values should be separated by comma
+buildfarm_platforms: "android,ios"
```

Comment:
`buildfarm_platforms` can be removed as they are now being set int the inventory